### PR TITLE
Found translations for all the missing Portuguese menu options

### PR DESCRIPTION
**Pull request by Blackweda**
This is a fix for issue #4938 

The menu for the Portuguese version of Brackets had a lot of missing information, so the menu items were set to the default representations in English. Not impossible to read, but messy and unprofessional. Anyway, I did the best I could with my basic French/Spanish grammar skills, and translated the Portuguese wording. 
Later this week I will get my friends to get the translation down even more, but for now, this should start the process of cleaning up the code for this issue. The entire Menu selection has been accounted for as well as the 'Working Files' section too!

### CODE REVIEW

File: src/config.json [hunk 1]
```diff
@@ -59,7 +59,7 @@
         "phantomjs": "1.9.18",
         "grunt-lib-phantomjs": "0.3.0",
         "grunt-eslint": "19.0.0",
-        "grunt-contrib-watch": "0.4.3",
```
Comment (swmitra) [inline src/config.json]: @Blackweda Sorry if I am missing something here, but is the change `"grunt-contrib-watch": "1.0.0"` is intentional for this PR ?

**Comment by Blackweda** [comment]
No, it kinda just showed up when I tried to push.. I'm new to github so I don't know if it was an updated master change that my repo didn't have.. I didn't know how to push without confirming it too...

**Comment by Blackweda** [comment]
Says it was a modified file even though I never touched it personally in my cloned files

File: src/config.json (hunk 1 above)
Comment (ficristo) [inline src/config.json]: It is a fallout of one of my PR...

File: samples/pt-pt/Primeiros Passos/index.html [hunk 2]
```diff
@@ -8,6 +8,7 @@
         <meta name="description" content="An interactive getting started guide for Brackets.">
         <link rel="stylesheet" href="main.css">
     </head>
+
```
Comment (ficristo) [inline samples/pt-pt/Primeiros Passos/index.html]: Could you revert this?

**Comment by ficristo** [comment]
@Blackweda is this ready to be merged? Were you able to find someone to review your translations?

**Comment by Blackweda** [comment]
Hey, I'm going to redo all this... I deleted my repo a few weeks ago so I'm gonna repeat everything again.

**Pull request closed without merging**